units/provisioning_jasper_restart.go

```diff
@@ -22,62 +21,64 @@ import (
 )
 
 const (
-	jasperDeployJobName    = "jasper-deploy"
-	expirationCutoff       = 7 * 24 * time.Hour // 1 week
-	jasperDeployRetryLimit = 50
+	jasperRestartJobName    = "jasper-restart"
```

Comment:
Are we concerned about existing jobs with the old name existing in the db erroring once we deploy this? If there's not a good reason to, we should probably keep the name

Reply:
I don't think so because this job should not have run yet - it only applies to hosts that have been up for at least a year, so it's really for static hosts and unexpirable spawn hosts.